docs/Home.md

```diff
@@ -2,41 +2,77 @@
+Quarkus is a Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, and it is also a ecosystem of ever growing extensions. In the beginning we were adding all extensions to the core Quarkus repository but eventually it became to much of overhead or just not always sensisble to include an extension into the core. 
+
+Some have since then created their own repositories external to Quarks, but others just never happeend because not everyone wants the overhead of maintaining a build setup while in other cases we actually had an idea for an extension but not solidified enough yet to be included in core.
+
+Quarkiverse is our proposed solution to create a "home" for such extensions/projects.
+
+#What is Quarkiverse
+
+This Quarkiverse github organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+
+Quarkus extensions hosted in the Quarkiverse organization will by default be included into the Quarkus extensions catalog displayed on [code.quarkus.io](http://code.quarkus.io) and the Quarkus command line tools (such as `mvn quarkus:list-extensions`, `gradle listExtensions`). To stay listed only requirement
+is that the extension keeps functioniong, stays uptodate and cause no harm.
 
 # Why Quarkiverse
 
 In the early days and actually up until recently, Quarkus extensions contributed by the community members (including the core Quarkus team) have been welcome in the Quarkus core repository at https://github.com/quarkusio/quarkus. Today the core repository has grown a lot and it's evident that it simply becomes troublesome to work with it for its contributors. That and other maintenance and infrastructure related issues inspired the creation of a separate organization called Quarkiverse to host Quarkus extension projects contributed by the community (including the Quarkus core team members themselves).
 
+The advantages to join Quarkiverse are:
+
+  . Automated and secured publishing of your maven releases to Maven Central.
+  . Automated Cross-testing of your extension with Quarkus builds/releases (see https://github.com/quarkusio/quarkus-ecosystem-ci)
+  . Inclusion in registry used by Quarkus tooling to browse extensions (i.e. code.quarkus, command line and IDE tools).
+  . Consistent formatting and release of project
+  . Quarkus team members can help and fix issues if project has build issues.
+  
+Note: While the Quarkus project has the "keys" to publish under `io.quarkiverse`, each project in Quarkiverse is driven and maintained by 
+the lead of that project. They decide what happens with the project - Quarkiverse will just expect releases to be compatible with Quarkus - if a project
+for some reason cannot do a release we can in the best scenario help or in the worst scenario we'll temporarily exclude the project from the registry to avoid
+end-users to have a bad experience.
```

Comment:
It's not like we are going to remove it. I don't think we will ever need to remove anything. If an extension is released it's added to the catalog for that Quarkus version. If a new version of Quarkus is released and the extension for some reason is not passing the tests with it we are not *adding* it to the new catalog for that new version of Quarkus. But it is also not removed from the catalog of the previous version of Quarkus which is still available for the users.

Reply:
I'm not fully following what is unclear here - we will/should remove extensions that doesn't work. 

I've made an update to say it can also just be a partial removal in case you are interpreting completely removal always.

Reply:
Are you talking about an extension that has zero releases and seems like it will never be released?
It depends on what you mean by remove. An extension may be compatible with versions of Quarkus in a specific version range and not compatible outside of that version range. Which should be reflected in the repository. In which case there is nothing to remove.

Reply:
FYI a prerequisite for an extension to be included in the catalog is to have at least one release.

Reply:
So what kind of removal are we talking about here?